lib/OpenQA/WebAPI/Controller/Admin/JobTemplate.pm

```diff
@@ -21,12 +21,22 @@ sub index {
     my ($self) = @_;
 
     my $schema = $self->schema;
-    $self->stash('group', $schema->resultset("JobGroups")->find($self->param('groupid')));
+    my $group  = $schema->resultset('JobGroups')->find($self->param('groupid'));
+    return $self->reply->not_found unless $group;
+
+    my $yaml = $group->template;
+    my $force_yaml_editor
+      = defined $yaml || $schema->resultset('JobTemplates')->search({group_id => $group->id}, {rows => 1})->count == 0;
+    $self->stash(
+        group             => $group,
+        yaml_templates    => $yaml,
```

Comment:
Just a minor concern but this should be singular. There's only one YAML document here given as a single string.

Reply:
fixed